acl-lib/dev/user.clj

```diff
@@ -1,12 +1,3 @@
-(ns user
```

Comment:
Does removing these :require, :use, and reset affect local dev at all?

Reply:
Nothing.  Stuff from a previous merge got it.  I will need to redo this branch.

Reply:
No.  My guess is that someone who was developing the acl lib used things in the user namespace, then got rid of the code in the user namespace, but never got rid of the require statements.